Directory.Build.targets

```diff
@@ -1,7 +1,10 @@
-﻿<Project>
+<Project>
   <PropertyGroup>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(MicrosoftNETCoreApp20PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
Can we drop the references to netcoreapp2.0 and 2.1?

Reply:
See [here](https://github.com/aspnet/Mvc/pull/7734#discussion_r186559595). Feel free to remove them yourself if it's appropriate for this repo.